libs/server/ACL/CommandPermissionSet.cs

```diff
@@ -167,10 +167,9 @@ public bool IsEquivalentTo(CommandPermissionSet other)
         /// </summary>
         private static ushort GetCommandListLength()
         {
-            int commandCount = (int)Enum.GetValues<RespCommand>().Where(static cmd => cmd != RespCommand.NONE && cmd != RespCommand.INVALID).Max();
-
-            int neededBits = commandCount;
-            int neededULongs = neededBits / 64;
+            // # of bits needed to represent all valid commands
+            var neededBits = (ushort)RespCommandExtensions.LastValidCommand + 1;
```

Comment:
consider using `CustomCommandManager.StartOffset`

Reply:
CustomCommandManager.StartOffset is also defined as (ushort)RespCommandExtensions.LastValidCommand + 1.
This is unrelated to custom commands, so it would be confusing to use CustomCommandManager in this context.